### DISCUSSION TITLE
Filter usability - nested field search results should be visible (expand group)

### DISCUSSION THREAD

**Discussion by jdjoshuadavison** [directus/directus#19706] [Feature Requests]
### The overarching problem
Users find the search and filters difficult to use.
This is one of a number of requests to help solve this problem.

### Nested field search results
When searching for a field in the filters UI, if the resulting field is inside a group, the group should be expanded and the non-matching fields in the group should be filtered away.

In the example below, I searched for a field called status. Several groups were returned. I expect those groups to be expanded automatically, and I don’t expect to see fields that don’t match my search term within the groups.

![filter-ui-field-search](https://github.com/directus/directus/assets/34007858/2b9e02e3-b06e-4457-a988-8186a22c6317)
![filter-ui-field-search-expanded](https://github.com/directus/directus/assets/34007858/9e0a6f5d-f162-4a99-83c0-27096e64c60c)

### Motivation

Users find the filters difficult to use.

### Detailed Design

N/A

### Must Have:
- Expand groups by default when searching for fields
- Filter fields within groups

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

None

### Adoption Strategy

Nothing to adopt - just enjoy a better UI!

**Comment by jdjoshuadavison** [comment]
Related #19707 
Related #19705

**Comment by rijkvanzanten** [comment]
This feels like a bug to me tbh. Search does work like the way you describe in other parts of the UI. Moving this to an issue instead 🙂

**Reply by rijkvanzanten** [reply]
We'll keep tracking this in https://github.com/directus/directus/issues/19710
